movie_REST_API/movie_REST_API/App/Builder/AssembleBuilder.swift

```diff
@@ -0,0 +1,53 @@
+// AssembleBuilder.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+protocol AssembleBuilderProtocol {
+    func makeMovieModule(router: RouterProtocol) -> UIViewController
+    func makeInfoMovieModule(router: RouterProtocol, movie: Movie) -> UIViewController
+}
+
+private enum Constants {
```

Comment:
```suggestion
private enum Constants {
```
в Нетворк сервис константы с url